app/Resources/assets/js/lsdoc/index.js

```diff
@@ -437,9 +437,110 @@ var SaltLocal = (function () {
         return false;
     }
 
+        var handlePdfFile = function (data) {
+           $.ajax({
+           url: data,
+           type: 'get',
+           dataType: 'json',
+            success: function (response) {
+            var pdfData = response;
+            var pdfMake = require('pdfmake');
+            var pdfFonts = require('pdfmake_fonts');
+            pdfMake.vfs = pdfFonts.pdfMake.vfs;
+    function getCFItemRelationship(identifier){
```

Comment:
Fixing the indentation within the entire `handlePdfFile` function would be good.  At first the getCFItemRelationship function looks like it is not part of the success function, similar to the rest of the content of the function.